### PR TITLE
Fix broken link in fr

### PR DESCRIPTION
**Pull request by remicres**
The documentation contains lot of dead links in the FR language.
I have noticed that the `{{< param "githubbranch" >}}` was in other portions of the doc, and seems outdated.
I have replaced every occurrence with "master".

### CODE REVIEW

**Comment by remicres** [comment]
I just have compared with the english doc, and enforce the same.

File: content/fr/docs/setup/release/building-from-source.md [hunk 1]
```diff
@@ -29,6 +29,6 @@ cd kubernetes
 make release
 ```
 
-Pour plus de détails sur le processus de release, voir le repertoire [`build`](http://releases.k8s.io/{{< param "githubbranch" >}}/build/) dans kubernetes/kubernetes.
+Pour plus de détails sur le processus de release, voir le repertoire [`build`](http://releases.k8s.io/master/build/) dans kubernetes/kubernetes.
```
Comment (Sea-n) [inline content/fr/docs/setup/release/building-from-source.md:32]: This file is not existed in English version.

Comment (Sea-n) [reply content/fr/docs/setup/release/building-from-source.md:32]: /hold

Comment (remicres) [reply content/fr/docs/setup/release/building-from-source.md:32]: > It's inconsistent with English version. /hold

Yes it is not consistent, but at least it points to an actual link.
This PR is just to fix the dead links, and do not intend to re-update the doc content.
The later could be done in another PR.

**Comment by Sea-n** [comment]
Okay, thanks for the fix! ✨ 
/unhold

**Pull request merged**